src/cli-validator/utils/processConfiguration.js

```diff
@@ -42,102 +42,112 @@ const validateConfigObject = function(configObject, chalk) {
   const allowedSpecs = Object.keys(defaultObject);
   const userSpecs = Object.keys(configObject);
   userSpecs.forEach(spec => {
-    if (!allowedSpecs.includes(spec)) {
-      validObject = false;
-      configErrors.push({
-        message: `'${spec}' is not a valid spec.`,
-        correction: `Valid specs are: ${allowedSpecs.join(', ')}`
-      });
-      return; // skip rules for categories for invalid spec
-    }
-
-    // check that all categories are valid
-    const allowedCategories = Object.keys(defaultObject[spec]);
-    const userCategories = Object.keys(configObject[spec]);
-    userCategories.forEach(category => {
-      if (!allowedCategories.includes(category)) {
+    // Do not check "spectral" spec rules
+    if (spec !== 'spectral') {
```

Comment:
This entire file is just indention changes with a single if statement, not sure why the github diff shows it so poorly